pkg/cli/scan.go

```diff
@@ -606,6 +607,18 @@ func resolveInputFileFromArg(inputFilePath string) (*os.File, error) {
 	}
 }
 
+// getRepositoryURL returns the URL of the APKINDEX.tar.gz file for the given
+// repository and architecture. If the repository URL already points to an
+// APKINDEX.tar.gz file, it will be returned as-is. User input may or may not
+// have included the architecture or the APKINDEX.tar.gz suffix, so construct
+// the full URL to provide better UX.
+func getRepositoryURL(repository, arch string) string {
```

Comment:
I'm not sure on this, since its a simple func to enhance the UX a bit on `scan` command, maybe we can move this to `resolveInputForRemoteTarget()`'s local-scope?

We can also consume this for `wolfictl apk ls`.

Reply:
We can come back to this after the PR